WordPress/Classes/ViewRelated/Domains/Views/DomainResultView.swift

```diff
@@ -0,0 +1,114 @@
+                .frame(height: subtitleHeight)
+
+                HStack(spacing: Metrics.noticeBoxHorizontalSpacing) {
+                    Image(uiImage: .gridicon(.infoOutline))
+                        .foregroundColor(Color(UIColor.muriel(color: .gray)))
+                        .accessibility(hidden: true)
+                    Text(TextContent.notice)
+                        .foregroundColor(Color(UIColor.muriel(color: .textSubtle)))
+                }.padding(Metrics.noticeBoxInsets)
+                .frame(maxWidth: .infinity)
+                .background(Color(UIColor.tertiaryFill))
+                .cornerRadius(Metrics.noticeBoxCornerRadius)
+            }.padding(Metrics.screenPadding)
+            VStack {
+                Spacer()
+                Button(action: dismiss) {
+                    ShapeWithTextView(title: TextContent.primaryButtonTitle).largeRoundedRectangle()
+                }
+            }.padding(Metrics.screenPadding)
+
+        }
+    }
+
+    private func subtitleWith(domain: String) -> NSAttributedString {
+        let templateString = String.localizedStringWithFormat(TextContent.subtitle, domain)
+        let attributedString = NSMutableAttributedString(string: templateString, attributes: [.font: Fonts.subtitleFont])
+        let range = (attributedString.string as NSString).localizedStandardRange(of: domain)
+        attributedString.setAttributes([.font: Fonts.subtitleFontBold], range: range)
+        return attributedString
+    }
+
+}
+
+// MARK: - Constants
+private extension DomainResultView {
+
+    private enum TextContent {
+        static let title = NSLocalizedString("freeToPaidPlans.resultView.title",
+                                             value: "All ready to go!",
+                                             comment: "Title for the domain purchase result screen. Tells user their domain was obtained.")
+        static let subtitle = NSLocalizedString("freeToPaidPlans.resultView.subtitle",
+                                                value: "Your new domain %@ is being set up.",
+                                                comment: "Sub-title for the domain purchase result screen. Tells user their domain is being set up.")
+        static let notice = NSLocalizedString("freeToPaidPlans.resultView.notice",
+                                              value: "It may take up to 30 minutes for your domain to start working properly",
+                                              comment: "Notice on the domain purchase result screen. Tells user how long it might take for their domain to be ready.")
+        static let primaryButtonTitle = NSLocalizedString("freeToPaidPlans.resultView.done",
+                                                          value: "Done!",
+                                                          comment: "Done button title on the domain purchase result screen. Closes the screen.")
+    }
+
+    private enum Metrics {
+        static let screenPadding = 15.0
```

Comment:
Nitpick. The padding should be `16`. Generally, it's a multiple of 4 (or 2).

<img width="490" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/f63583d6-d59a-498e-986a-c3ce694752e7">

You can check other metrics as well, they are all likely to be multiple of 4s, including the corner radius.

Reply:
I'm glad you flagged this, I went back and updated the spacings and sizes to match the Figma design.